modules/admission-webhook-about.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-about_{context}"]
+= Admission webhooks
+
+In addition to {product-title} default admission controller plug-ins, admission webhooks enable you to introduce admission plug-ins invoked from webhook servers over HTTP at defined endpoints.
+
+There are two types of configurable admission webhooks in {product-title}:
+
+//Future xref - * During the admission process, a mutating admission controller can invoke xref:../architecture/admission-controllers.adoc#mutating-admission-webhooks[mutating admission webhooks] to perform tasks, such as injecting affinity labels.
```

Comment:
an adminssion controller invokes webhooks? Confusing. The webhooks are plugins themselves.

Reply:
This has been reworded as per our conversation today.